### PR TITLE
Use `classic-level` to build and extract compendium packs

### PR DESCRIPTION
**Pull request by In3luki**
This needs a followup PR to remove the `.db` endings from all `packs/data/` folders.

- Building works
- Extraction of specific packs works
- Extraction of all packs works

Issues:
- The packs are not shown in the compendium sidebar. Probably related to our override. `game.packs.get("pf2e.abomination-vaults-bestiary").render(true)`  works.
- Existing `*.db` files in `static/packs` need to be removed manually (one time) or they will be copied to `dist`.

### CODE REVIEW

File: build/build-packs.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const packDirPaths = fs.readdirSync(packsDataPath).map((dirName) => path.resolve
 // Loads all packs into memory for the sake of making all document name/id mappings available
 const packs = packDirPaths.map((p) => CompendiumPack.loadJSON(p));
 const documentCounts = packs.map((p) => p.save());
-const total = documentCounts.reduce((total, c) => total + c, 0);
+const total = (await Promise.all(documentCounts)).reduce((total, c) => total + c, 0);
```
Comment (stwlam) [inline build/build-packs.ts:13]: move the `Promise.all` up a line

File: build/extract-packs.ts [hunk 2]
```diff
@@ -24,9 +23,9 @@ const args = argv
                 default: true,
             })
             .example([
-                ["npm run $0 spells.db /path/to/foundryvtt/Config/options.json"],
-                ["npm run $0 spells.db C:\\Users\\me\\this\\way\\to\\options.json"],
-                ["npm run $0 spells.db # copy of config at ./foundryconfig.json or otherwise using dist/"],
+                ["npm run $0 spell /path/to/foundryvtt/Config/options.json"],
```
Comment (stwlam) [inline build/extract-packs.ts:26]:
```suggestion
                ["npm run $0 spells /path/to/foundryvtt/Config/options.json"],
```

File: build/lib/extractor.ts [hunk 3]
```diff
@@ -93,9 +93,6 @@ class PackExtractor {
                 foundryPacks.map(async (filePath) => {
                     const dbFilename = path.basename(filePath);
 
-                    if (!dbFilename.endsWith(".db")) {
-                        throw PackError(`Pack file is not a DB file: "${dbFilename}"`);
-                    }
                     if (!fs.existsSync(filePath)) {
                         throw PackError(`File not found: "${dbFilename}"`);
                     }
```
Comment (stwlam) [inline build/lib/extractor.ts:98]: Is this a directory now?

**Comment by In3luki** [comment]
Updated. Should build now.

**Pull request merged**